esignet-service/src/main/resources/application-local.properties

```diff
@@ -74,6 +74,9 @@ mosip.esignet.captcha.site-key=test-site-key
 mosip.esignet.signup-id-token-expire-seconds=1800
 mosip.esignet.signup-id-token-audience=mosip-signup-oauth-client
 
+mosip.esignet.host=localhost
+mosip.esignet.claims.schema.url=classpath:/verified_claims_request_schema.json
```

Comment:
This should also be part of application-default.properties file

Reply:
added